src/css/base.scss

```diff
@@ -70,7 +70,7 @@
 
 .form-container {
   border: 2px solid green;
-  height: 30%;
+  height: 50%;
   margin: 10%;
   display: flex;
   flex-direction: column;
```

Comment:
Once you get towards the end of Iteration 3, what do you think about having the display:flex and flex-directions becoming variables in a separate .scss file?